sites/funmooc/src/backend/templates/richie/base.html

```diff
@@ -20,3 +20,69 @@
 {% endblock topbar_contact %}
 
 {% block body_footer_title %}{% endblock body_footer_title %}
+
+{% block body_js %}
+{{ block.super }}
+<script type="text/javascript" src="//tag.aticdn.net/602676/smarttag.js"></script>
+<script>
+
+    function getTag(){
+
+        var tag = new ATInternet.Tracker.Tag({ site: 602676, secure: true});
+
+        var url = new URL(window.location.href)
```

Comment:
As you are using `url` in the rest of the code, use `URL` interface is useless here. As you already do the job and that `URL` is may be still pretty young to be used without a polyfill, do not use it here.
You could just use `location`
- `location.pathname`
- `location.search`
-...

Reply:
FIY : `URL` interface has a `searchParams` property to access the individual query parameters found in search.